### PR TITLE
Feat/login page

### PR DESCRIPTION
**Pull request by sewilee**

### CODE REVIEW

File: client/assets/scss/colors.scss [hunk 1]
```diff
@@ -1,4 +1,45 @@
-$blue: rgba(52, 88, 170, 1);
-$grey: rgba(106, 106, 106, 1);
-$red: rgba(204, 0, 0, 1);
-$white: rgba(255, 255, 255, 1);
\ No newline at end of file
+//Instructions:
```
Comment (clee421) [inline client/assets/scss/colors.scss]: Hmm, so this PR is depending on another PR. Remember to rebase afterwards.

File: client/src/components/authentication/login/login.tsx [hunk 2]
```diff
@@ -26,22 +27,26 @@ const Login: React.FC = (): ReactElement => {
...
-          <span>password</span>
-          <input {...password} />
-        </label>
-      </div>
-
-      <div className="submit-container">
-        <button type="button" data-test="submit-btn" onClick={handleSubmit}>
-          Submit
-        </button>
+    <div className="auth-container">
```
Comment (clee421) [inline client/src/components/authentication/login/login.tsx:27]: I saw the layout for it. What do you think about adding a `Form` component?

File: client/src/index.scss [hunk 3]
```diff
@@ -1,3 +1,5 @@
+@import '../assets/scss/colors.scss';
```
Comment (clee421) [inline client/src/index.scss]: Project has global imports, you should be able to just import `@import 'colors';` instead of using a relative path

Comment (clee421) [reply client/src/index.scss]: I see what you're doing with the color now. Let's remove this along with the bottom code.

File: client/src/index.scss [hunk 4]
```diff
@@ -6,13 +8,18 @@ html {
...
+}
+
+h1, h2, h3, h4, h5, h6, body, p, span {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  color: $primary-text;
```
Comment (clee421) [inline client/src/index.scss]: If you look at `index.tsx` this was meant to be used as a reset file. so let's not set anything here. It's meant for resetting.

File: client/src/index.tsx [hunk 5]
```diff
@@ -3,7 +3,7 @@ import ReactDOM from "react-dom";
 
 // CSS Reset
 // import '@csstools/normalize.css';
```
Comment (clee421) [inline client/src/index.tsx]: Does your styling break if you uncomment this line? This was commented out as part of the migration over to using a node server and webpack and I forgot to comment it back in.

File: client/src/components/authentication/login/login.tsx (hunk 2 above)
Comment (sewilee) [reply client/src/components/authentication/login/login.tsx:27]: Talked offline. Will add in another branch

**Review by clee421 — APPROVED** [review]

**Comment by clee421** [comment]
LGTM 🚢

**Pull request merged**